Dockerfile

```diff
@@ -67,7 +67,7 @@ RUN apt-get update -y -q && \
     apt-get install -y clang-3.9 lldb-3.9 && \
     apt-get install -y libc6-i386  libsuitesparse-dev && \
     export LLVM_CONFIG=/usr/lib/llvm-3.9/bin/llvm-config && \ 
-    git clone --recursive https://github.com/simonsfoundation/CaImAn.git && \
+    git clone --recursive https://github.com/ecobost/CaImAn.git && \
```

Comment:
I did but I don't know whether they'll merge it soon or solve it in a different way. We can let the original for now if you want and hope they do merge it or I'll wait until they send us the dendrite version and then fork and change that version